README.md

```diff
@@ -155,7 +155,8 @@ dependencies {
     }
 
     // diktat ruleset
-    ktlint("org.cqfn.diktat:diktat-rules:1.0.1") {
+    ktlint("org.cqfn.diktat:diktat-rules:1.0.3-SNAPSHOT") {
```

Comment:
This is a usage instruction for possible users, therefore it should use released version which can be obtained from maven repo. Versions with `SNAPSHOT` suffix are accessible only if you build from source. Also, this line is duplicated.